test/TestApps/PushNotificationsTestApp/PushNotificationsTestApp/main.cpp

```diff
@@ -0,0 +1,20 @@
+// PushNotificationsTestApp.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+
+int main()
+{
+    std::cout << "Hello World!\n";
```

Comment:
Whats this for? Should it be checked in?

Reply:
IF it's for automation, I would recommend that you put it in as part of a seperate PR